common/app/facebookimages/openGraphOverlay.scala

```diff
@@ -0,0 +1,20 @@
+package facebookimages
+
+import model.Article
+import views.support.{FacebookOpenGraphImage, ImgSrc}
+
+
+object openGraphOverlay {
+  def overlayImage(article: Article): Article = {
```

Comment:
Should this method name hints it is returning an article. As is it looks more like it is returning an image.